src/main/java/appeng/items/tools/powered/powersink/AEBasePoweredItem.java

```diff
@@ -82,6 +84,12 @@ public boolean isBarVisible(ItemStack stack) {
         return true;
     }
 
+    @Override
+    public boolean allowNbtUpdateAnimation(Player player, InteractionHand hand, ItemStack oldStack,
+            ItemStack newStack) {
+        return getAECurrentPower(oldStack) == getAECurrentPower(newStack);
```

Comment:
I wasn't sure if any of the tool items had any other property aside from power that would cause an update animation to occur. If not, then I suppose `false` would do instead.

Reply:
Yeah idk, don't we return `false` in things like the portable cell?

Reply:
In specific tool classes like portable cells this is indeed overridden to return `false`. I forgot however if this is the case with the colour applicator since I assume that could have its animation go off when switching to a different colour.